song-server/src/main/java/bio/overture/song/server/model/analysis/Analysis.java

```diff
@@ -82,6 +85,17 @@
   @JsonIgnore
   private AnalysisData analysisData;
 
+  @NotNull
+  @Builder.Default
+  @ToString.Exclude
+  @EqualsAndHashCode.Exclude
+  @OneToMany(
+      mappedBy = ModelAttributeNames.ANALYSIS,
+      cascade = CascadeType.ALL,
+      orphanRemoval = true,
+      fetch = FetchType.EAGER)
```

Comment:
@andricDu Any issue with this Eager load? It shouldn't have performance issues and is generally relevant info that is being requested in this ticket.

Reply:
`EAGER` is prone to the N+1 problem.

Reply:
For me I would change it to `LAZY` simply because of SONG's usage patterns, a tiny addition in query time gets multiplied a hundred fold. 

I'll let @rtisma decide on this one as he knows the performance pains better than I do.

Reply:
I'm pushing an update that moves this to lazy fetch, so not an issue.